views/stats.html

```diff
@@ -0,0 +1,57 @@
+{% extends 'layouts/base.html' %}
+
+{% block main %}
+<article class="result">
+  <div class="stats-node">
+    <h3>Number of trials: <span id="trialsCount">{{stats.trialsCount}}</span></h3>
+  </div>
+
+  <div class="stats-node">
+    <h3>Number of trials per registry:</h3>
+    <ul class="meta" id="trialsPerRegistry">
+      {% for trialPerRegistry in stats.trialsPerRegistry %}
+      <li>
+        <dt>{{ trialPerRegistry.registry }}</dt>
+        <dd>{{ trialPerRegistry.count }}</dd>
+      </li>
+      {% endfor %}
+    </ul>
+  </div>
+
+  <div class="stats-node">
+    <h3>Number of trials per year:</h3>
+    <ul class="meta" id="trialsPerYear">
+      {% for trialPerYear in stats.trialsPerYear %}
+      <li>
+        <dt>{{ trialPerYear.year }}</dt>
+        <dd>{{ trialPerYear.count }}</dd>
+      </li>
+      {% endfor %}
+    </ul>
+  </div>
+
+  <div class="stats-node">
+    <h3>Top 10 locations with most trials:</h3>
+    <ul class="meta" id="topLocation">
+      {% for topLocation in stats.topLocations %}
+      <li>
+        <dt>{{ topLocation.name }}</dt>
+        <dd>{{ topLocation.count }}</dd>
+      </li>
+      {% endfor %}
+    </ul>
+  </div>
+
+  <div class="stats-node">
+    <h3>Date when we updated each registry: </h3>
+    <ul class="meta" id="updatedRegistry">
+      {% for dateRegistry in stats.dateRegistry %}
+      <li>
+        <dt>{{ dateRegistry.name }}</dt>
+        <dd>{{ dateRegistry.updatedate | formatDate }}</dd>
```

Comment:
This seems to be a residue from old code, as you've changed `updatedate` to `latest_update_date`